app/src/main/res/layout/tray.xml

```diff
@@ -12,14 +12,14 @@
     </data>
 
     <RelativeLayout
-        android:layout_width="@dimen/tray_width"
-        android:layout_height="@dimen/tray_height"
+        android:layout_width="wrap_content"
```

Comment:
Ditto integrating the UI changes in the try in a single commit with a detailed explanation